docs/reference/ml/df-analytics/apis/delete-dfanalytics.asciidoc

```diff
@@ -19,12 +19,8 @@ Deletes an existing {dfanalytics-job}.
 [[ml-delete-dfanalytics-prereq]]
 == {api-prereq-title}
 
-If the {es} {security-features} are enabled, you must have the following 
-built-in roles or equivalent privileges:
-
-* `machine_learning_admin`
-
-For more information, see <<built-in-roles>> and {ml-docs-setup-privileges}.
+It is required to have the `manage_ml` cluster privilege to use this API. This 
```

Comment:
If you want to align with the style of the introductory sentence, I think this also works:
```suggestion
Requires the `manage_ml` cluster privilege. This 
```